backends/ubpf/ubpfTable.cpp

```diff
@@ -200,6 +153,53 @@ UBPFTable::UBPFTable(const UBPFProgram *program,
     actionList = table->container->getActionList();
 
     setTableSize(table);
+    setTableKind();
+}
+
+void UBPFTable::emitInstance(EBPF::CodeBuilder *builder) {
+    cstring keyTypeStr;
```

Comment:
This code looks very much like UBPFRegister. It would be nice to have a helper function implement both.

Reply:
Done.